### ISSUE TITLE
testool improvement

### ISSUE THREAD

**Issue by hero78119** [privacy-scaling-explorations/zkevm-circuits#1303]
### Describe the feature you would like

- [x] add hyperlink on reporting page therefore click can navigate to respective ethereum-tests source file. Link need to attach with respective submodule version
- [x] fix blank path in tabular data
- [x] add a "light" mode to only re-run failed/panic cases since last test
- [ ] decouple reading tests and compiling bytecode src.  Lazily compile bytecode src that really needed.

**Comment by ChihChengLiang** [comment]
@hero78119 Is this task completed after #1306?

**Comment by hero78119** [comment]
> @hero78119 Is this task completed after #1306?

Thanks for the reminding.
The first 3 items already finished. The forth one still pending. I just update the ckeckedbox

**Comment by hero78119** [comment]
Closed, and raise separate issue #1574 to solve point 4
